backend/database/init.sql

```diff
@@ -63,49 +76,100 @@ create table if not exists combined_risk (
     liquefaction_risk boolean not null default false
 );
 
-/*create table if not exists soft_story_addresses (
+create table if not exists soft_story_properties (
     identifier integer not null,
     block varchar(255),
     lot varchar(255),
     parcel_number varchar(255),
     property_address varchar(255),
-    address varchar(255),
+    address varchar(255) not null,
     tier integer,
     status varchar(255),
     bos_district integer,
-    --point: Mapped[Geometry] = mapped_column(Geometry("POINT", srid=4326))
+    point Geometry(point, 4326),
     sfdata_as_of timestamp,
-    sfdata_loaded_at timestamp
-    --update_timestamp timestamp
+    sfdata_loaded_at timestamp,
+    update_timestamp timestamp
 );
-*/
+
+insert into addresses (eas_fullid,        address,      unit_number,  address_number, street_name, street_type, parcel_number, block, lot, cnn,    longitude,  latitude, zip_code,  point,                                                  supdist,                    supervisor, supname,         nhood,     sfdata_as_of,             created_timestamp,      update_timestamp) values 
```

Comment:
Why do we have both (longitude, latitude), and point? Granted, since this table is probably going to be short-lived, it probably doesn't matter. Just curious